bundle/language_en.properties

```diff
@@ -1176,6 +1176,8 @@ angal.patient.note
 angal.patient.parenttogether                                                                           = Parents are together
 angal.patient.patientbrowser.title                                                                     = Patient Browser
 angal.patient.patientphoto                                                                             = Patient Photo
+angal.patient.patientphoto.error.title                                                                 = Patient Photo Error
+angal.patient.patientphoto.error.filenotfound.msg                                                      = Patient photo file was not found
```

Comment:
this change belongs to [OP-879](https://openhospital.atlassian.net/browse/OP-879).

All other langues should not be modified